include/acl_hal_mmd.h

```diff
@@ -33,6 +33,21 @@ typedef struct {
   unsigned long long size;  /* size of this memory */
 } aocl_mmd_memory_info_t;
 
+// Interface to simulator to describe streaming kernel arguments that are
+// excluded from the invocation image. Streaming arguments are passed to the
+// simulator by calling aocl_mmd_simulation_streaming_kernel_info(), before
+// writing the kernel invocation image containing non-streaming arguments.
+struct aocl_mmd_streaming_kernel_arg_info_t {
+  // unique identifier for the bus-functional model (BFM)
+  std::string name;
+  // argument value
+  std::vector<char> value;
+};
+
+struct aocl_mmd_streaming_kernel_info_t {
+  std::vector<aocl_mmd_streaming_kernel_arg_info_t> args;
+};
```

Comment:
The API is broken either way if the struct is modified, so maybe remove the wrapper?
```suggestion
using aocl_mmd_streaming_kernel_info_t = std::vector<aocl_mmd_streaming_kernel_arg_info_t>;
```